src/linter.js

```diff
@@ -438,7 +438,10 @@ export default class Linter {
       })
       .then(() => {
         this.print();
-        return;
+        /* istanbul ignore if  */
```

Comment:
This is skipped in the code coverage because the test runs against un-instrumented code.